scalardl-test/verification-config.toml

```diff
@@ -24,6 +24,8 @@
   is_verification = true
   num_accounts = 10
   population_concurrency = 32
+  retry_interval_milli_sec = 1000
+  max_retries_for_read = 10
```

Comment:
I apologise for inconsistent naming of variables. I will ensure they are named consistently. The read failure occurs on readBalances function. The retry and interval sleep delay was added on this function because this is the first function called. I will update the variable names and make it specific.

Reply:
> The read failure occurs on readBalances function
This is probably because readBalances is the first operation, right?

Reply:
Sorry for the delay.

> This is probably because readBalances is the first operation in the recovery, right?

yes. It is the first operation.

>It's not very bad to retry readBalances only but I think it's more solid to retry the whole recovery process?

I am not sure on this , so I will check this also by adding a retry on the whole process and provide an update regarding this.

Reply:
I tried adding a repeat for the entire execute function and it requires additional code added which may be unnecessary and redundant and my implementation was not good enough. It may be possible to add it in an efficient way but currently configuring intervals as per the initial suggestion may be suitable in this scenario.